brtnext/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
Kan dit niet in de htaccess als redirect?
De JS in deze pagina is valide, maar maakt het lastiger om met een Nonce te gaan werken in de context van CSP headers.